app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -92,6 +93,12 @@ AccountService = ($state, $auth, $modal, $http, ShortFormApplicationService) ->
       return
     )
 
+  Service.getMyApplications = ->
```

Comment:
it's a good question... I have some things like the private method in `account_controller.rb` called `current_user_applications` however everything else I kind of used the "my applications" wording. I guess I liked the semantic nature of it, "get my applications" describes what it's doing (from the user's perspective) and lines up with the name of the page. But I agree we should make sure we have consistent naming conventions as we add more user features. Like do we call other features "my {xxx}" and "get my {xxx}"...